### ISSUE TITLE
Drying up swagger_api_root

### ISSUE THREAD

**Issue by bizmurr** [fotinakis/swagger-blocks#9]
Is there a way we can dry up `swagger_api_root`?

Defining:

```
key :swaggerVersion, '1.2'
key :apiVersion, '1.0.0'
key :basePath, ...
```

everytime, feels a bit messy.

It would be more convenient if you could define it one place and extend it/override it. Any thoughts on how to do this?

**Comment by fotinakis** [comment]
How I currently do this in a Rails app is with an initializer and a constant, like:

```ruby
# initializers/swagger_setup.rb:
SWAGGER_CONFIG = {
  swagger_version: '1.2',
  api_version: 'v1',
  # TODO: make this dynamic based on the current host.
  base_path: 'http://localhost:3000/api/v1',
}.freeze
```

```ruby
# controllers
swagger_api_root :users do
  key :swaggerVersion, SWAGGER_CONFIG[:swagger_version]
  key :apiVersion, SWAGGER_CONFIG[:api_version]
  key :basePath, SWAGGER_CONFIG[:base_path]
  key :resourcePath, '/users'
end
```

It looks like they've addressed this in Swagger 2.0 :+1: because the spec is now a single file and the root object has a single `swagger` key. So as soon as #8 lands, you'll be able to DRY this up with an upgrade to the new spec, so I'm going to close this in preference for Swagger 2.0.

**Comment by bizmurr** [comment]
Perfect, thanks! :+1:
